src/material/ItemStack.ts

```diff
@@ -10,11 +10,15 @@ export class ItemStack implements ToJava {
         return new ItemStack(_itemstack);
     }
 
+    public static fromJavaNullable(_itemstack: Java.Value): ItemStack | null {
+        return _itemstack == null ? null : this.fromJava(_itemstack);
+    }
```

Comment:
For dumb JavaScript reasons, you need to update `this.fromJava(_itemstack)` to instead be `ItemStack.fromJava(_itemstack)`

Somewhere else in the code (I forget where), you do something like this:
```ts
someArray.map(ItemStack.fromJavaNullable)
```
and when you do that, `fromJavaNullable` is called with an undefined receiver, so `this` will not work.